### PR TITLE
update kintohub.json

### PR DESCRIPTION
**Pull request by nandiheath**
# Pull request motivation(s)

### What is the current behaviour?

Click on the search content is go to legacy pages and shows 404

### What is the expected behaviour?

Should go to the latest pages.

### CODE REVIEW

File: configs/kintohub.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "index_name": "kintohub",
   "start_urls": [
-    "https://docs.kintohub.com/docs/getting-started.html",
-    "https://docs.kintohub.com/docs/"
+    "https://docs.kintohub.com/docs/introduction/getting-started"
```
Comment (s-pace) [inline configs/kintohub.json]: https://docs.kintohub.com/docs/introduction/getting-started will be the only page crawled and thus available from the search.

Comment (nandiheath) [reply configs/kintohub.json]: thank you for the advice. and i added the sitemap as well.

File: configs/kintohub.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
```
Comment (s-pace) [inline configs/kintohub.json]:
```suggestion
    "https://docs.kintohub.com/docs/"
```

`start_urls` will be used as a regex to match the sitemap in this context. The broader it is, the better

Comment (s-pace) [reply configs/kintohub.json]: image
I can't do it on my own.

Comment (nandiheath) [reply configs/kintohub.json]: ok, just fixed it

**Pull request merged**